src/helpers/codeGenerator.ts

```diff
@@ -11,41 +11,58 @@ import { EventType } from '../constants';
  * Helper functions that handle each action type.
  * @param event
  */
+ async function getPatternMethod(): Promise<string> {
+  const defaultConfig = { pattern: 'css selectors' }; // 默认配置
+  return new Promise((resolve) => {
+    chrome.storage.local.get(defaultConfig, (items) => {
```

Comment:
need error handling as well. This function should likely be extracted as the same logic is used in `eventRecorder.ts`